### PR TITLE
Auto-update typed envelopes version metadata

### PR DESCRIPTION
**Pull request by zblach**
Also removed CTIX.

### CODE REVIEW

**Comment by zblach** [comment]
one of the things I also fixed was the files inclusion in `package.json`. I had a leading '.' which made directory resolution incorrect.

File: src/common.ts [hunk 1]
```diff
@@ -81,7 +80,7 @@ export class AADMeta implements Datagram<AADMeta> {
   deserialize(data: Uint8Array): AADMeta | null {
     const buf = { bs: data };
     const metadataVersion = _decoder.decode(extractVarintPrefixed(buf));
-    if (metadataVersion !== AADMeta.METADATA_VERSION) {
+    if (metadataVersion !== AADMETA_VERSION) {
```
Comment (zblach) [inline src/common.ts:83]: this would actually make it incompatible with artifacts generated by the `0.1.0` release. I'll be modifying this momentarily.

**Review by amilich — APPROVED** [review]
No comments, LGTM :)

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@skiff-org/typed-envelopes",
-  "version": "0.1.0",
+  "version": "0.1.1",
   "license": "AGPL-3.0",
   "author": "skiff-org",
   "main": "./dist/index.js",
```
Comment (jaismith) [inline package.json:6]: does this still need the `./`?

File: src/index.ts [hunk 3]
```diff
@@ -0,0 +1,3 @@
+export * from './common';
```
Comment (jaismith) [inline src/index.ts:1]: switching to manual generation?

**Review by jaismith — COMMENTED** [review]
lgtm! small question about `./` in `"main"`, and are we switching to manual index files?

File: src/index.ts (hunk 3 above)
Comment (zblach) [reply src/index.ts:1]: yeah. After I migrated the jest configuration from a .js to a .ts file, `ctix` started including it as a part of the indexing, and doesn't seem to obey its own 'exclude' directives, so I dropped that build step and just checked in these generated files. 

I can revisit this once I start working on the other box implementations, but I don't imagine that these index files are going to be changing much until then.

File: package.json (hunk 2 above)
Comment (zblach) [reply package.json:6]: I don't see the harm in keeping this the same. I don't think it's the same as the `gitignore`-style includes later on in this file. I've imported this package into a sandbox TS project and the import paths looked sane, and that was really the driving force behind ctix + this packaging stuff.


Screen Shot 2021-04-20 at 1 41 20 PM

**Pull request merged**